patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/README.md

```diff
@@ -0,0 +1,87 @@
+# Dedicated Server Utils
+
+This directory contains utilities for managing a dedicated ***Linux*** server. There is the `ManageServer.sh` script for directly managing an installation or a Docker container for automatic management.
+
+## Using the script
+
+The `ManageServer.sh` script can be used to install tML either directly from the Github release or from SteamCMD. The script is made to run fully standalone, so just download it to your server and run it. No other files from the repo are needed.
+
+Github installations can be done using `./ManageServer.sh --install --github`. SteamCMD is the default so just leave out `--github` to use it instead. By default Github installations will be in `~/tModLoader`, and SteamCMD installations will be in `~/Steam/steamapps/common/tModLoader`.
+
+During installation, [enabled.json](#install.txt-and-enabled.json), [install.txt](#install.txt-and-enabled.json), and any `.tmod` files will be checked for in the directory of the script or in the directory specified by `--checkdir`.
+
+Note that for Github installations `unzip` must be on your `PATH`, and for SteamCMD installations `SteamCMD` must be on your `PATH`. Install it from your package manager or from [steam](https://developer.valvesoftware.com/wiki/SteamCMD).
+
+For additional information, run `./ManageServer.sh --help` to view the available arguments.
+
+## Using the Docker container
+
+Before building and running the container first you need to prepare a directory to pass to the container. This applies to `docker` and `docker-compose` This directory will link to `~/.local/share/Terraria` inside of the container. Within the directory, place your [enabled.json](#install.txt-and-enabled.json), [install.txt](#install.txt-and-enabled.json), `serverconfig.txt`, and any `.tmod` files. This directory will be used in the `docker run` step. To manually add worlds, place them in `tModLoader/Worlds` inside of your directory (you may have to create the directory yourself).
+
```

Comment:
This needs an intro for people unfamiliar with docker. Should they be using docker? Why is it not an option as part of `ManageServer.sh`? Is it less fragile? Installing steam_cmd is hard.

There needs to be a section at the very top to help people figure out which method they're meant to use, why pick steamcmd vs docker vs github.